evals/prompt/base.py

```diff
@@ -12,7 +12,7 @@
 ENCODER_LOCK = threading.Lock()
 
 # This is an approximation to the type accepted as the `prompt` field to `openai.Completion.create` calls
-OpenAICreatePrompt = Union[str, list[str], list[int], list[list[int]]]
+OpenAICreatePrompt = Union[str, list[str]]
```

Comment:
I think we shouldn't change this since this is specifically for `openai.Completion.create`